### PR TITLE
Case view

### PR DESCRIPTION
**Pull request by ayushi-siddhu**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ const routes = [
     name: 'Settings',
     component: Settings
   },
+  {
+     path: '/caseview',
+     name: 'CaseView',
+     component: CaseView
```
Comment (chayan007) [inline src/router/index.js:38]: Did you import `CaseView`? This should throw error for sure.

File: src/views/CaseView.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
+    <div class="container">
+        <div class="row justify-content-md-around">
+          <CaseViewPage></CaseViewPage>
+        </div>
+      </div>
+</template>
+
+<script>
+// @ is an alias to /src
```
Comment (chayan007) [inline src/views/CaseView.vue:10]: remove comment !

File: src/views/CaseView.vue [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+import CaseViewPage from '../components/CaseViewPage.vue';
```
Comment (chayan007) [inline src/views/CaseView.vue:12]: a line gap please.

Comment (chayan007) [reply src/views/CaseView.vue:12]: also use `@` instead of `..`

**Review by chayan007 — CHANGES_REQUESTED** [review]
1. CaseViewPage to be pulled inside the CaseView. Hence, CaseViewPage (which is a component) needs to be deleted.
2. Page Design has serious flaws. Please coordinate with @itisdb (one of them - I can see the navbar and page elements merged together.)

**Review by raju-max — APPROVED** [review]

File: src/views/CaseView.vue (hunk 2 above)
Comment (ayushi-siddhu) [reply src/views/CaseView.vue:10]: done

File: src/views/CaseView.vue (hunk 3 above)
Comment (ayushi-siddhu) [reply src/views/CaseView.vue:12]: ok

**Review by raju-max — APPROVED** [review]

**Review by chayan007 — APPROVED** [review]

**Pull request merged**